### ISSUE TITLE
Autosort

### ISSUE THREAD

**Issue by flash1293** [flash1293/ekofe#133]
Learn from previous move-actions how to sort a list (new entry in the list-context-menu)

**Comment by bnfinet** [comment]
When an item is "resurrected" from "Last Recently Used" it would be nice if it remembered its prior order.  Is that what you mean?

**Comment by flash1293** [comment]
No, the idea here is to learn the order of commonly occurring items over time. Could you create a new issue for your suggestion? I like it a lot and it’s an easy addition to the current state.
